### PR TITLE
Peer programed with erdem and andrew

### PR DESCRIPTION
**Pull request by christylaguardia**
#### Single-line Summary
**Today, Christy, Erdem and(rew) Andrew paired together. It took about 4 hours**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we added our connection strings
  2. Next, in server.js, we added the SELECT query in the app.get('/articles' ... function
  3. Then, we added the INSERT statements in the app.post('/articles' ...  function
  4. Then, we added the UPDATE statements in the app.put('/articles/:id'... function
  5. Then, we added the DELETE statements in the app.delete('/articles/:id'... function
  6. Then, we tested the .insertRecord, .updateRecord and .deleteRecord functions.
  7. Finally, Ken helped us fix our bugs, and some bugs with the starter code.

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick'). 
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: starter-code/server.js [hunk 1]
```diff
@@ -45,17 +46,37 @@ app.get('/articles', function(request, response) {
...
+        ),
+        $2, $3, $4, $5)
+        ON CONFLICT DO NOTHING
+      `,
+    // DONE: Add the required values from the request as data for the SQL query to interpolate
+      [
+        request.body.author,
+        request.body.title,
+        request.body.category,
+        request.body.publishedOn, // QUESTION: shouldn't this be today's date?
```
Comment (kbrumer) [inline starter-code/server.js]: What is the client sending?

**Comment by kbrumer** [comment]
👍

File: starter-code/server.js [hunk 2]
```diff
@@ -45,17 +46,37 @@ app.get('/articles', function(request, response) {
...
     client.query(
-      `Thing1`,
-      [Thing2]
+      `INSERT INTO articles (author_id, title, category, publishedOn, body)
+      VALUES (
+        (
+          SELECT author_id
+          FROM authors
+          WHERE authors.author = $1
+        ),
```
Comment (JPGothro) [inline starter-code/server.js]: Nice use of SELECT to get the author_id used in the insert of the article.

File: starter-code/server.js (hunk 1 above)
Comment (JPGothro) [inline starter-code/server.js]: ANSWER: Isn't the publishedOn only populated if the check box is checked on the form? If not checked, this is null, correct?

**Pull request closed without merging**